roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb

```diff
@@ -20,6 +20,7 @@
   end
 end
 
+{{ if inventory_hostname in groups['controller'] }}
```

Comment:
I believe these should be `{% if inventory_hostname in groups['controller'] %}`
and 
`{% endif %}`